scripts/test_nearlib.sh

```diff
@@ -16,14 +20,13 @@ git clone https://github.com/near/create-near-app.git
 # Make sure to use local nearcore for tests
 export NODE_ENV=local
 
-SRC_DIR=$(pwd)/$(dirname "${BASH_SOURCE[0]}")
-export HOME=$SRC_DIR/../testdir
+export HOME=$(readlink -e "$REPO_DIR/../testdir")
 
 # Run near-api-js tests
-cd near-api-js
+pushd "${REPO_DIR}"/near-api-js
```

Comment:
Again, `cd` is fine and also this change isn’t necessary.

Reply:
Ok, I changed it back to `cd`.

This makes reading the code easier, as I don't need to follow, in which current directory we are at the moment.